src/main/java/com/iotiq/application/messages/orderedproduct/OrderedProductDto.java

```diff
@@ -0,0 +1,26 @@
+package com.iotiq.application.messages.orderedproduct;
+
+import com.iotiq.application.domain.Price;
+import com.iotiq.application.messages.customer.contact.BasicInfo;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+@Getter
+@Setter
+public class OrderedProductDto {
+
+    private UUID productId;
+    private String title;
+    private Price price;
+    private String imageUrl;
+    private String orderNumber;
+    private LocalDateTime orderDate;
```

Comment:
Can you check if using `LocalDateTime` is appropriate here? Does it have timezone? Does it need timezone? @mesutiotiq

Reply:
We saved utc date in order.java

Reply:
How do you make sure the conversion between LocalDateTime and the UTC equivalent is correct?

Reply:
resolved in:
https://github.com/Iotiq-Erste/kimono-application/commit/0035546235394f98e33501b4cde44d10f803c2cf